build/quarks-secret/_vendir/deploy/helm/quarks-secret/templates/cluster-role.yaml

```diff
@@ -0,0 +1,103 @@
+{{- if .Values.global.rbac.create }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: {{ template "quarks-secret.fullname" . }}
+rules:
+- apiGroups:
+  - certificates.k8s.io
+  resources:
+  - certificatesigningrequests
+  verbs:
+  - approve
+  - create
+  - delete
+  - get
+  - list
+  - update
+  - watch
+
+- apiGroups:
+  - certificates.k8s.io
+  resources:
+  - signers
+  resourceNames:
+  - kubernetes.io/legacy-unknown
+  verbs:
+  - approve
+- apiGroups:
+  - certificates.k8s.io
+  resources:
+  - certificatesigningrequests/approval
+  verbs:
+  - create
+  - update
+
+
+- apiGroups:
```

Comment:
Could we make this permission conditional based on the applyCRD value as well? It seems like the operator-based CRD creation flow is the only motivation for the permission. It worked on our cluster when we removed the permission and redeployed from scratch with the CRD resource included in the rendered yaml.

Reply:
Ah, very nice. :+1:

Reply:
pulled in a new chart where this issue is resolved.